docs/backend-api/resources/tracking/tracker/alarm_mode.md

```diff
@@ -46,17 +55,23 @@ Changes the state of alarm mode of device. The device must be online.
 
 | name | description | type | format |
 | :------ | :------ | :----- | :----- |
-| tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked | int | 999199 |
-| enabled | True if alarm mode should be enabled | boolean | true/false |
+| tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. | int | 999199 |
+| enabled | True if alarm mode should be enabled. | boolean | true/false |
 
-#### example
+#### examples
 
-```abap
-$ curl -X POST '{{ extra.api_example_url }}/tracker/alarm_mode/set' \
--H 'Content-Type: application/json' \ 
--d '{"tracker_id": "999199", "enabled": "true", "hash": "a6aa75587e5c59c32d347da438505fc3"}'
+=== cURL
```

Comment:
Tabs should have this syntax:

````
=== "cURL"

    ```shell
    curl -X POST '{{ extra.api_example_url }}/tracker/apn_settings/read' \
        -H 'Content-Type: application/json' \ 
        -d '{"tracker_id": "123456", "hash": "a6aa75587e5c59c32d347da438505fc3"}'
    ```

=== "HTTP GET"

    ```
    {{ extra.api_example_url }}/tracker/apn_settings/read?tracker_id=123456&hash=a6aa75587e5c59c32d347da438505fc3
    ```
````

See https://squidfunk.github.io/mkdocs-material/reference/content-tabs/

Reply:
ready